Automattic/woocommerce-payments
# Fix deprecated strtolower on order received page

Fixes #8232

## Changes proposed in this Pull Request









* Make sure order currency is initialized in the `FrontendCurrencies` class by the time the decimal point formatting function is called.

#### Testing instructions





1. Make sure your store is running on PHP 8.1 or newer.
2. Place an order.
3. Open the Order Received page.
4. Make sure there are no deprecation warnings.
5. Go to **My Account &rarr; Orders** and view one of the orders.
6. Make sure there are no deprecation warnings. 

-------------------


**Post merge**
